doc/source/reference/changelog.rst

```diff
@@ -12,6 +12,353 @@ The `CREDITS file <https://github.com/yt-project/yt/blob/master/CREDITS>`_
+    - added support for dust fields and derived fields 
+      `PR 2387 <https://github.com/yt-project/yt/pull/2387>`__.
+    - added support for derived fields for hydro runs 
+      `PR 2381 <https://github.com/yt-project/yt/pull/2381>`__.
+    - API documentation and docstrings for AMRVAC frontend 
+      `PR 2384 <https://github.com/yt-project/yt/pull/2384>`__, 
+      `PR 2380 <https://github.com/yt-project/yt/pull/2380>`__, 
+      `PR 2382 <https://github.com/yt-project/yt/pull/2382>`__.
+    - testing-related PRs for AMRVAC: 
+      `PR 2379 <https://github.com/yt-project/yt/pull/2379>`__, 
+      `PR 2360 <https://github.com/yt-project/yt/pull/2360>`__.
+    - add verbosity to logging of geometry or `geometry_override` 
+      `PR 2421 <https://github.com/yt-project/yt/pull/2421>`__.
+    - add attribute to `_code_unit_attributes` specific to AMRVAC to ensure 
+      consistent renormalisation of amrvac datasets. See 
+      `PR 2357 <https://github.com/yt-project/yt/pull/2357>`__.
+    - parse AMRVAC's parfiles if user-provided  
+      `PR 2369 <https://github.com/yt-project/yt/pull/2369>`__.
+    - enssure that min_level reflects dataset that has refinement 
+      `PR 2475 <https://github.com/yt-project/yt/pull/2475>`__.
+    - fix derived unit parsing  `PR 2362 <https://github.com/yt-project/yt/pull/2362>`__.
+    - update energy field to be `energy_density` and have units of code 
+      pressure  `PR 2376 <https://github.com/yt-project/yt/pull/2376>`__.
+- Support for the AdaptaHOP halo finder code 
+  `PR 2385 <https://github.com/yt-project/yt/pull/2385>`__.
+- yt now supports geographic transforms and projections of data with 
+  cartopy with support from `PR 1966 <https://github.com/yt-project/yt/pull/1966>`__.
+- annotations used to work for only a single point, they now work for multiple points 
+  on a plot, see `PR 2122 <https://github.com/yt-project/yt/pull/2122>`__.
+- cosmology calculations now have support for the relativistic energy density of the 
+  unvierse, see `PR 1714 <https://github.com/yt-project/yt/pull/1714>`__. 
+  This feature is accessible to cosmology datasets and was added to the Enzo frontend. 
+- the eps writer now allows for arrow rotation. this is accessible with 
+  the ``rotate`` kwarg in the ``arrow`` function. 
+  See `PR 2151 <https://github.com/yt-project/yt/pull/2151>`__.
+- allow for dynamic load balancing with parallel loading of timeseries 
+  data using the ``dynamic`` kwarg. `PR 2149 <https://github.com/yt-project/yt/pull/2149>`__.
+- show/hide colorbar and show/hide axes are now available for 
+  ``ProfilePlot`` s. These functions were also moved from the PlotWindow to the 
+  PlotContainer class. `PR 2169 <https://github.com/yt-project/yt/pull/2169>`__.
+- The angular momentum has been reversed compared to previous versions of yt. 
+  This may cause breaking changes in your code. 
+  See `PR 2043 <https://github.com/yt-project/yt/pull/2043>`__.
```

Comment:
This has a potentially major impact on users. Can we promote it some how? Make a separate category of breaking changes or something like that?

Reply:
Ok i've moved it up to a section right after the intro to the release. Does that work?